src/main/java/carpet/script/language/DataStructures.java

```diff
@@ -238,9 +238,12 @@ else if (lv.size() == 2)
             for (int i = 1; i < lv.size(); i++)
             {
                 if (!(container instanceof ContainerValueInterface)) return (cc, tt) -> Value.NULL;
-                container = ((ContainerValueInterface) container).get(lv.get(i).evalValue(c));
```

Comment:
Please don't fix in here, but in NbtValue where it implements .get - otherwise this code will be executing every time you need for example to get the value of a list like `lst:1`. This issue affects ONLY nbts

Reply:
Also this will then also affect all other usages of CVI.get in all other cases. Check NBTSerializableValue#580